core/src/test/java/org/elasticsearch/discovery/DiscoveryWithServiceDisruptionsIT.java

```diff
@@ -1086,14 +1085,49 @@ public void testIndicesDeleted() throws Exception {
         assertAcked(prepareCreate("test"));
         ensureYellow();
 
-        String masterNode1 = internalCluster().getMasterName();
+        final String masterNode1 = internalCluster().getMasterName();
         NetworkPartition networkPartition = new NetworkUnresponsivePartition(masterNode1, dataNode.get(), getRandom());
         internalCluster().setDisruptionScheme(networkPartition);
         networkPartition.startDisrupting();
         internalCluster().client(masterNode1).admin().indices().prepareDelete("test").setTimeout("1s").get();
```

Comment:
BTW, changing the `PUBLISH_TIMEOUT_SETTING` to 0 throws the following exception:
FailedToCommitClusterStateException[timed out while waiting for enough masters to ack sent cluster state. [1] left]